src/layouts/partial-cards.html

```diff
@@ -253,15 +276,16 @@
     return window.hassUtil.isComponentLoaded(hass, 'introduction');
   },
 
-  computeViews: function (hass) {
+  computeAllViews: function (hass) {
     var views = window.HAWS.extractViews(hass.states);
-
     // If default view present, it's in first index.
-    if (views.length > 0 && views[0].entity_id === 'group.default_view') {
-      views.shift();
+    if (views.length > 0 && views[0].entity_id === this.DEFAULT_VIEW_ENTITY_ID) {
+      this.defaultView = views.shift();
+    } else {
+      this.defaultView = null;
     }
 
-    return views;
+    this.views = views;
```

Comment:
Wouldn't that be less efficient, because the array generated by `window.HAWS.extractViews` into a variable would need to be copied by `slice` into `views`?